frontend/src/components/Customer/TrackOrder.tsx

```diff
@@ -139,18 +171,57 @@ function OrderDetailsCard(){
             <Typography gutterBottom variant="h5" component="h2">
                 Order Details
             </Typography>
-            <OrderItemsTable></OrderItemsTable>
+            <OrderItemsTable>(orders)</OrderItemsTable>
             </CardContent>
             </Card>
         </Container>
     );
 }
 
 export default function TrackOrder() {
+  const [orders, setOrders] = React.useState<Array<Order>>([]);
```

Comment:
Similarly copy pasted from Employees/Order.tsx. No need to add duplication. This track order is supposed to get only one order for a given customer who just completed an order. Why are we pulling every Order from the backend and then displaying it to the customer? 

We should be hitting the /orders/:id endpoint with the given order id that has just been completed instead. This will have to be passed through to TrackOrder from a higher level component.